doc/content/microreactors/hpmr_h2/simba_model.md

```diff
@@ -0,0 +1,129 @@
+
+\begin{equation}
+  \frac{\partial {c_H}}{\partial t} = \nabla \cdot [-D(\nabla {c_H} + \frac{Q {c_H}}{R T^2} \nabla T)],
+\end{equation}
+
+where D is the diffusion coefficient of hydrogen in $YH_x$, R is the gas constant, T is the temperature, and Q is the heat of transport . In the Bison input, the diffusion coefficient is set to one in order to accelerate the convergence for the steady-state calculations given than the asymptotic hydrogen distribution does not depend on the diffusion coefficient [!citep](Terlizzi2023). The Bison model is coupled to 101 heat pipes (HP) sub-applications that model the flow of sodium within the reactor. The 101 instances of HPs are generated in the correct position through the MultiApps system from the single heat pipe input described in the following section. The coupling between Bison and Sockeye is performed through convective boundary conditions.
+
+The full input is reported below for the sake of completeness.
+
+!listing microreactors/hpmr/steady/thermal_ss.i language=cpp
+
+The Sockeye input contains the model for a single heat pipe. The latter uses the VOFM to model the 1D sodium flow within the core region of the HP and a 2D heat conduction model in the annulus and wick region.  The full input is reported below for the sake of completeness.
+
+!listing microreactors/hpmr/steady/heatpipe_vapor_only.i language=cpp
+
+
+## Multiphysics Coupling
+
+[simba_mpmap] shows the computational workflow for the coupled simulation. The left box includes the preliminary operations—namely, the mesh preparation and cross-section tabulations performed with Serpent (v. 2.1.32).
+
+!media hpmr_h2/simba_mpmap.jpeg
+    caption= Integrated computational workflow [!citep](Terlizzi2023).
+    id=simba_mpmap
+    style=width:80%; margin-left:auto; margin-right:auto
+
+Griffin is run to simulate the neutron transport in the core. The power density, $P_d$, computed from the neutron flux, is then transferred to Bison that is utilized to solve for the temperature in the reflector, $T_r$, moderator, $T_m$, and fuel, $T_f$, and the hydrogen-yttrium stoichiometric ratio in the yttrium hydride, here denoted as $c_H$. Bison is coupled to 101 single heat pipes sub-applications that are used to compute the equivalent heat transfer coefficient, $h_{eq}$, and the secondary-side temperature, $T_{secondary}^i$, to obtain the thermal fluids' response. The updated temperature field and the hydrogen-yttrium stoichiometric ratio are then fed back into Griffin, where the macroscopic cross sections are updated based on the new operating conditions,
+
+
+## Running the Model
+
+To run the input via the HPC, load the following modules:
+
+```language=CPP
+module load use.moose moose-apps direwolf
+```
+
+Finally, to run the input, make sure you are in the correct directory, then use the run command below. The input being ran will be the neutronics file.
```

Comment:
no you need a submission script, cant run this on the head node
Maybe add a line for an interactive session?

Reply:
line for interactive session added